crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -1393,6 +1394,43 @@ impl From<&Indices> for IndexFormat {
     }
 }
 
+bitflags! {
+    /// Mesh pipeline key bits start from the highest bit and go downward. PBR
+    /// mesh pipeline key bits start from the lowest bit and go upward. This
+    /// allows the PBR bits in the downstream crate `bevy_pbr` to coexist in the
+    /// same field without any shifts.
```

Comment:
Two small things:
1. The line break leaving the word `PBR` on the first line is a bit unfortunate, I read it at least 3 times before I realized it wasn't typoed
2. I don't know how many key bits we have left, but since they're split and we now rely on them being OR'able it might make sense to make a small test (or possibly const assert) that they fit? This might prevent a headache when someone adds the final bit that makes one of them overflow.

Reply:
I reworded the comment and added a `const_assert_eq!` that the bits didn't collide.